.github/workflows/build.yml

```diff
@@ -253,7 +253,7 @@ jobs:
           fail-on-cache-miss: true
           enableCrossOsArchive: true
       - name: "Download OpenTofu CLI package"
-        uses: actions/download-artifact@9bc31d5ccc31df68ecc42ccf4149144866c47d8a # v3.0.2
+        uses: actions/download-artifact@fa0a91b85d4f404e444e00e005971372dc801d16 # v4.1.8
```

Comment:
`nitpicking`
Why not using directly `@v4.1.8?`
https://github.com/actions/download-artifact/releases/tag/v4.1.8

Same for the others

🤔

Reply:
Honestly, I just followed an existing approach. Theoretically, it could be re-tagged to point to some other commit and then break the CI, but I believe this is pretty low risk. If you don't mind, I will merge this as is :)